modules/gateway/variables.tf

```diff
@@ -48,13 +48,19 @@ variable "gateway_allowed_address_prefixes" {
   default     = []
 }
 
-# Application gateway specific configurations
+# Application gateway specific configurations 
 
 variable "gateway_enable_private_access" {
   description = "Enable or disable private access to the application gateway"
   type        = bool
 }
 
+variable "disable_agw" {
+  description = "Disables the creation of application gateway from the terraform module."
```

Comment:
of Application Gateway since that's the name of the component in Azure

Reply:
This comment is still valid

Reply:
Fixed